model_armor/create_template.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def create_model_armor_template(project_id: str, location: str, template_id: str):
+    # [START modelarmor_create_template]
+
+    from google.cloud import modelarmor_v1
+    client = modelarmor_v1.ModelArmorClient(transport="rest", client_options={
```

Comment:
Can you use the `ClientOptions` class? Similar to this sample https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/discoveryengine/search_sample.py